src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -46,7 +46,9 @@ class Yaml
      *
      * @throws ParseException If the YAML is not valid
      *
-     * @deprecated The ability to pass file names to Yaml::parse() was deprecated in 2.2 and will be removed in 3.0. Please, pass the contents of the file instead.
+     * @deprecated since version 2.2, to be removed in 3.0.
```

Comment:
This change is wrong. The new message looks like the whole method is deprecated, which is absolutely not the case (using the ``@deprecated`` tag is already an issue btw, because static analysis tools wil report all method usages as deprceated, but this should be fixed in older versions).
Can you revert this message ?